site/presentations.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Presentations
+layout: content
+---
```

Comment:
Yes, that's standard practice for all Markdown converters (in this case Jekyll) to have a YAML preamble with metadata